tests/dummy/app/pods/demo/overflow/template.hbs

```diff
@@ -0,0 +1,57 @@
+{{! BEGIN-SNIPPET overflow }}
+{{frost-modal-info-message
+  confirm=(hash
+    text='99'
+  )
+  details=(component 'list-em'
+    things=bottles
+  )
+  isVisible=isModalVisible
+  title='How many bottles?'
+  onClose=(action 'stopCounting')
+}}
+{{! END-SNIPPET }}
+
+<div class='frost-modal-demo-api'>
+  <div class='frost-modal-demo-title'>
+    Notes
+  </div>
+  <div class='frost-modal-demo-notes'>
+    <p>
+      I can has more content?
```

Comment:
Yeah, there is a bit of a theme here where I'm trying to make the demo writing a bit more interesting